README.md

```diff
@@ -263,44 +246,90 @@ Whenever static files are changed, the CMS may need to be manually told to serve
+
+### Test Coverage
+
+Coverage is sent to codecov on commits to the repo (see Github Actions for branch to see branch coverage). Ideally we only merge positive code coverage changes to `main`.
+
+### Production Deployment
+
+The Core CMS runs in a Docker container as part of a set of services managed with Docker Compose.
+
+CMS images are built by [Jenkins](https://jenkins01.tacc.utexas.edu/view/WMA%20CEP/job/Core_CMS/) and published to the [Docker Hub repo](https://hub.docker.com/repository/docker/taccwma/core-cms).
+
+To update the CMS in production or dev, the corresponding [Core Portal Deployments](https://github.com/TACC/Core-Portal-Deployments) env file should be updated with a tag matching an image previously built and published to the taccwma/core-cms repo.
+
+Deployments are initiated via [Jenkins](https://jenkins01.tacc.utexas.edu/view/WMA%20CEP/job/Core_Portal_Deploy/) and orchestrated, tracked, and directed by [Camino](https://github.com/TACC/Camino) on the target server.
+
+
+## Deployment Steps
+
+1. Build and publish portal image with [Jenkins](https://jenkins01.tacc.utexas.edu/view/WMA%20CEP/job/Core_CMS/)
+2. Update deployment settings, particularly the `CMS_TAG` environment variable in [Core Portal Deployments](https://github.com/TACC/Core-Portal-Deployments) with new tag name
+3. Deploy new image with [Jenkins](https://jenkins01.tacc.utexas.edu/view/WMA%20CEP/job/Core_Portal_Deploy/)
+
+
+## Contributing
+
+### Development Workflow
+
+We use a modifed version of [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) as our development workflow. Our [development site](https://dev.cep.tacc.utexas.edu) (accessible behind the TACC Network) is always up-to-date with `main`, while the [production site](https://prod.cep.tacc.utexas.edu) is built to a hashed commit tag.
+- Feature branches contain major updates, bug fixes, and hot fixes with respective branch prefixes:
+    - `task/` for features and updates
+    - `bug/` for bugfixes
+    - `fix/` for hotfixes
+
+### Best Practices
+
+Sign your commits ([see this link](https://help.github.com/en/github/authenticating-to-github/managing-commit-signature-verification) for help)
+
+### Resources
+
+* [Learn Markdown](https://bitbucket.org/tutorials/markdowndemo)
+* [Tapis Project (Formerly Agave)](https://tacc-cloud.readthedocs.io/projects/agave/en/latest/)
 
-Testing is manual review of build output. From within `core_cms` container, run `npm run build` from the repository root directory, then review `taccsite_cms/static/site_cms/css/build/_test.css`.
 
-## Reference
+<!-- Link Aliases -->
 
-- [Formatting & Linting](https://confluence.tacc.utexas.edu/x/HoBGCw)
-- [DjangoCMS Stand Alone Site](https://confluence.tacc.utexas.edu/x/G4G-Ag)
+[Core Portal Deployments]: https://github.com/TACC/Core-Portal-Deployments
+[Camino]: https://github.com/TACC/Camino
+[Core CMS]: https://github.com/TACC/Core-CMS
```

Comment:
Maybe add a link to Core-CMS-Resources and Core-Styles for completeness?

Reply:
They will come in https://github.com/TACC/Core-CMS/pull/442 when I edit this README to actually link to https://github.com/TACC/Core-CMS-Resources/ and https://github.com/TACC/Core-CMS-Resources/. (This README, as is, does not link to either repo.)